`cargo check --no-default-features --features=s3` doesn't compile on master

Hello 👋 

On master (commit 9569396) command `cargo check --no-default-features --features=s3` fails with

```
error[E0433]: failed to resolve: use of undeclared crate or module `chrono`
   --> src/util.rs:514:44
    |
514 | impl<Tz: chrono::TimeZone> DateTimeExt for chrono::DateTime<Tz>
    |                                            ^^^^^^ use of undeclared crate or module `chrono`

error[E0433]: failed to resolve: use of undeclared crate or module `chrono`
   --> src/util.rs:514:10
    |
514 | impl<Tz: chrono::TimeZone> DateTimeExt for chrono::DateTime<Tz>
    |          ^^^^^^ use of undeclared crate or module `chrono`

For more information about this error, try `rustc --explain E0433`.
error: could not compile `sccache` due to 2 previous errors
```

--- a/src/util.rs
+++ b/src/util.rs
@@ -510,7 +510,7 @@ pub trait DateTimeExt {
     fn to_rfc7231(&self) -> String;
 }
 
-#[cfg(any(feature = "azure", feature = "gcs", feature = "s3"))]
+#[cfg(any(feature = "azure", feature = "gcs"))]
 impl<Tz: chrono::TimeZone> DateTimeExt for chrono::DateTime<Tz>
 where
     Tz::Offset: core::fmt::Display,

